extensions/shared/external-media/sources/with-media.js

```diff
@@ -7,12 +7,13 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import { speak } from '@wordpress/a11y';
 import apiFetch from '@wordpress/api-fetch';
+import { withNotices, Modal } from '@wordpress/components';
 import { createHigherOrderComponent } from '@wordpress/compose';
+import { select } from '@wordpress/data';
```

Comment:
What do you think to compose with select instead of requiring it?

```suggestion
import { select } from '@wordpress/data';
import { withSelect } from '@wordpress/data';
```